libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -1336,9 +1348,12 @@ int HTTPClient::handleHeaderResponse()
                         break; // We found a match, stop looking
                     }
                 }
+                continue;
```

Comment:
Is this continue correct?

Reply:
yes, because the only thing after this line is to check whether the headerline is the "empty line" aka whether we're at the end of the section of headers. if we have determined that there is a ':' in the current line we already know it is no longer empty and can stop caring about the empty line case.